build-logic-commons/gradle-plugin/src/main/kotlin/gradlebuild.reproducible-archives.gradle.kts

```diff
@@ -18,6 +18,8 @@
 tasks.withType<AbstractArchiveTask>().configureEach {
     isPreserveFileTimestamps = false
     isReproducibleFileOrder = true
+    @Suppress("DEPRECATION")
     dirMode = Integer.parseInt("0755", 8)
+    @Suppress("DEPRECATION")
```

Comment:
❓ Shouldn't we dogfood the new API in our build instead of suppressing deprecation?